src/CounterInterceptor/MyWorkflow.workflow.cs

```diff
@@ -0,0 +1,56 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Workflows;
+
+[Workflow]
+public class MyWorkflow
+{
+    private string name = string.Empty;
+    private string title = string.Empty;
+    private bool exit; // automatically defaults to false
+
+    [WorkflowRun]
+    public async Task<string> ExecAsync()
+    {
+        // wait for greeting info
+        await Workflow.WaitConditionAsync(() => name != null && title != null);
+
+        // Execute Child Workflow
+        string result = await Workflow.ExecuteChildWorkflowAsync(
+            (MyChildWorkflow wf) => wf.ExecChildAsync(name, title),
+            new()
+            {
+                Id = Constants.ChildWorkflowId,
+            });
```

Comment:
```suggestion
            new() { Id = Constants.ChildWorkflowId });
```

Don't have to span lines

Reply:
Fixed.